spikes/lk_qdac/main.cpp

```diff
@@ -0,0 +1,97 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#include "PinNames.h"
+
+#include "drivers/HighResClock.h"
+#include "rtos/ThisThread.h"
+
+#include "CoreI2C.h"
+#include "CoreQDAC.h"
+#include "HelloWorld.h"
+#include "LogKit.h"
+
+using namespace leka;
+using namespace std::chrono;
+
+//
+// MARK: - Global definitions
+//
+
+auto corei2c = CoreI2C {PinName::SENSOR_PROXIMITY_MUX_I2C_SDA, PinName::SENSOR_PROXIMITY_MUX_I2C_SCL};
+auto dac	 = CoreQDACMCP4728 {corei2c, 0xC0};
+
+auto data = uint16_t {};
+
+void printInputRegistersAndMemory()
+{
+	const auto channels = std::array<Channel, 4> {Channel::A, Channel::B, Channel::C, Channel::D};
+	log_info("Print !\n");
+	for (Channel ch: channels) {
+		data = dac.read(ch);
+		log_info("Channel %d", ch);
+		log_info("Input Registers");
+		log_info("Data : %x", data);
+
+		data = dac.read(ch, true);
+		log_info("EEPROM Memory");
+		log_info("Data : %x", data);
+		log_info("\n");
```

Comment:
Je propose quelque chose de plus condensé encore: 
```suggestion
		log_info("================= Channel %d ================= ", ch);
		data = dac.read(ch);
		log_info("Input Registers === Data: %x", data);
		data = dac.read(ch, true);
		log_info("EEPROM Memory ===== Data: %x", data);
```